### ISSUE TITLE
The callback of `ActionSheet` on Android receives a string index

### ISSUE THREAD

**Issue by boringer** [GeekyAnts/NativeBase#1353]
## react-native, react and native-base version

- React Native: 0.49.3.
- React: 16.0.0-beta.5.
- Native Base: 2.3.3.

## Expected behaviour

Type of index should be `number`, like iOS.

## Actual behaviour

Is `string`.

## Steps to reproduce (code snippet or screenshot)

```js
ActionSheet.show({
    options: [
        // ...
    ]
}, (index) => {
    console.log(typeof index); // => 'string'
});
```

## Is the bug present in both ios and android or in any one of them?

On Android.

**Comment by akhil-geekyants** [comment]
Made a PR #1370 for this issue

**Comment by akhil-geekyants** [comment]
#1370

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)
